bootstrap-salt.sh

```diff
@@ -2352,7 +2374,7 @@ install_debian_6_deps() {
     fi
 
     # shellcheck disable=SC2086
-    wget $_WGET_ARGS -q http://debian.saltstack.com/debian-salt-team-joehealy.gpg.key -O - | apt-key add - || return 1
+    __fetch_verify http://debian.saltstack.com/debian-salt-team-joehealy.gpg.key 267d1f152d0cc94b23eb4c6993ba3d67 3100 | apt-key add - || return 1
```

Comment:
This line belongs to `install_debian_6_deps()` function. Debian 6 (old old stable) reached [end-of-life](https://wiki.debian.org/LTS) recently. So the entire function should be dropped along with adding Squeeze to outdated releases.